pkg/plugins/resources/githubrelease/main.go

```diff
@@ -1,46 +1,56 @@
 package githubrelease
 
 import (
-	"path"
-
 	"github.com/mitchellh/mapstructure"
-	"github.com/updatecli/updatecli/pkg/core/tmp"
 	"github.com/updatecli/updatecli/pkg/plugins/scms/github"
 	"github.com/updatecli/updatecli/pkg/plugins/utils/version"
 )
 
+// Spec defines a specification for a "gittag" resource
+// parsed from an updatecli manifest file
+type Spec struct {
+	Owner         string         // Owner specifies repository owner
```

Comment:
Not blocker for this PR, but as recently discussed, It would be good to put comment before a struct field and not on the same line.

Reply:
Yep, that would be a separated (and strictly scoped) PR to avoid messing up our rebases ;) But thanks to remind it to us 👍